DQM/SiStripCommissioningDbClients/src/LatencyHistosUsingDb.cc

```diff
@@ -137,14 +137,18 @@ bool LatencyHistosUsingDb::update( SiStripConfigDb::DeviceDescriptionsRange devi
   for( CommissioningHistograms::Analysis it = data().begin(); it!=data().end();++it) {
     if(dynamic_cast<SamplingAnalysis*>( it->second ) &&
        dynamic_cast<SamplingAnalysis*>( it->second )->granularity()==sistrip::PARTITION       )
+      {
       anal = dynamic_cast<SamplingAnalysis*>( it->second );
       latency = uint16_t(ceil(anal->maximum()/(-25.)))>latency ? uint16_t(ceil(anal->maximum()/(-25.))) : latency;
```

Comment:
This appears to be a bug because otherwise latency is set  by the last thing anal=it->second points to.